lib/usd/translators/mayaReferenceUpdater.cpp

```diff
@@ -48,14 +49,21 @@
 
 namespace {
 
+// Clear the auto-edit flag on a USD Maya Reference so that it does not
+// get edited immediately again. Clear in all variants, since each
+// variant has its own copy of the flag.
 void clearAutoEdit(const UsdPrim& prim)
 {
-    if (prim.IsValid()) {
+    UsdPrim parentPrim = prim.GetParent();
+    MAYAUSD_NS::applyToAllVariants(parentPrim, true, [prim]() {
```

Comment:
We normally just use the actual namespace name instead of the define. So just MayaUsd::